README.md

```diff
@@ -69,7 +69,18 @@ Install devel packages for `SDL2` and `SDL2_mixer`.\
 Compile with CMake; tested with GCC 10, Clang 11.\
 To cross-compile for Windows, install a 64-bit version of mingw and its `SDL2` and `SDL2_mixer` distributions, then use the `mingwcc.cmake` toolchain.
 
-[![Packaging status](https://repology.org/badge/tiny-repos/spacecadetpinball.svg)](https://repology.org/project/spacecadetpinball/versions) 
+```sh
+apt update
+sudo apt install libsdl2-dev libsdl2-mixer-dev
+
+cd ../path/to/SpaceCadetPinball
+mkdir build && cd build
+cmake .. -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++  # gcc
+cmake .. -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++  # clang
```

Comment:
cmake chooses the compiler automatically; most people don't need to care about specifying gcc/clang. If you think it's absolutely necessary, I think having a small section later saying that these parameters exist would be better.

Reply:
your changes in https://github.com/k4zmu2a/SpaceCadetPinball/pull/242/commits/8fd9f41dfb3e4d4b08cef42a2af9140009aa6672 look good 👍🏻